src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/Controllers/TableBuilderControllerTests.cs

```diff
@@ -34,12 +38,88 @@ public async Task Query()
         {
             var tableBuilderService = new Mock<ITableBuilderService>();
 
-            tableBuilderService.Setup(s => s.Query(_releaseId, _query)).ReturnsAsync(
-                new TableBuilderResultViewModel
+            tableBuilderService
+                .Setup(s => s.Query(_query, default))
+                .ReturnsAsync(new TableBuilderResultViewModel
+                    {
+                        Results = new List<ObservationViewModel>
+                        {
+                            new()
+                        }
+                    }
+                );
+
+            var controller = BuildTableBuilderController(tableBuilderService: tableBuilderService.Object);
+            var result = await controller.Query(_query);
+
+            Assert.IsType<TableBuilderResultViewModel>(result.Value);
+            Assert.Single(result.Value.Results);
+        }
+        
+        [Fact]
+        public async Task Query_CancellationToken()
```

Comment:
Is it actually worth testing this? Seems a bit redundant if we know that the attribute works correctly in its own tests. Same for `Query_ReleaseId_CancellationToken`